src/main/java/org/jenkinsci/plugins/workflow/steps/durable_task/DurableTaskStep.java

```diff
@@ -85,7 +91,7 @@ public String getEncoding() {
     }
 
     @DataBoundSetter public void setEncoding(String encoding) {
-        this.encoding = encoding;
+        this.encoding = Util.fixEmpty(encoding);
```

Comment:
Does not handle blank strings 🐜

Reply:
Sorry, what?

> Convert empty string to null.

That is exactly what I want here.

Reply:
(anyway for #64)